File: TeamspeakBot/TeamspeakBot/ts3_sdk_3.0.4.4/examples/client_android/app/src/main/java/com/teamspeak/ts3sdkclient/eventsystem/IEventListener.java
package com.teamspeak.ts3sdkclient.eventsystem;

/**
 * TeamSpeak 3 sdk client sample
 *
 * Copyright (c) 2007-2017 TeamSpeak-Systems
 *
 * @author Anna
 * Creation date: 08.02.17
 *
 * Eventmethod Interface for ts3clientlibrary events
 * All classes that want to receive ts3client library events have to implement this Interface
 */
public interface IEventListener {
    void onTS3Event(IEvent event);
}

File: TeamspeakBot/TeamspeakBot/ts3_sdk_3.0.4.4/examples/client_android_javasound/app/src/main/java/com/teamspeak/ts3sdkclient/eventsystem/IEventListener.kt
package com.teamspeak.ts3sdkclient.eventsystem

/**
 * TeamSpeak SDK client sample
 *
 * Copyright (c) 2007-2018 TeamSpeak-Systems
 *
 * Eventmethod Interface for ts3clientlibrary events
 * All classes that want to receive ts3client library events have to implement this Interface
 */
interface IEventListener {
    fun onTS3Event(event: IEvent)
}
